.changeset/silent-nails-hide.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Update the Table component's onSortBy method signature.
```

Comment:
I wonder if this sentence is redundant since the same information is repeated in the more detailed explanation below?

```suggestion
Updated the Table component's `onSortBy` method signature.
```

Reply:
I think I kind of treated these as commit messages, with a heading and body. Is it fine to have a single, longer sentence as a changeset?

Reply:
Yes, that's preferable since the changeset will be rendered as one bullet point.

Reply:
Done in b68d861